src/main/java/biblioteket/roborally/programcards/Card.java

```diff
@@ -36,31 +39,38 @@ public void doCardAction(IRobot robot, IBoard board) {
                 robot.turnLeft();
                 break;
             case MOVE_1:
-                robot.moveForward(board);
+                robot.move(robot.getDirection(), board);
                 break;
             case MOVE_2:
-                robot.moveForward(board);
-                if (robot.getPlayer().hasLivesLeft())
+                robot.move(robot.getDirection(),board);
+                //if (robot.getPlayer().hasLivesLeft())
```

Comment:
Lag en issue på dette isteden for å kommentere det ut, eventuelt se om du klarer å fikse det.